examples/mic_aggregator/README.md

```diff
@@ -0,0 +1,187 @@
+# PDM Microphone Aggregator Example
+
+This example provides a bridge between 16 PDM mics to either TDM16 slave or USB Audio demo running on the explorer board. It uses a modified mic_array with multiple decimator threads to support 16 DDR mics on a single 8b input port. The example is written in 'bare-metal' and runs directly on the XCORE device without an RTOS.
+
+The decimators are configured to 48kHz PCM output. The 16 channels are loaded into a 16 slot TDM slave peripheral running at 24.576MHz bit clock or a USB Audio Class 2 asynchronous interface and are optionally amplified. The TDM build provides a simple I2C slave interface to allow gains to be controlled at run-time.
+
+For the TDM build, a simple TDM16 master is included as well as a local 24.576MHz clock source so that mic_array and TDM16 slave may be tested standalone through the use of jumper cables. These may be removed when integrating into a system with TDM16 master supplied.
+
+Obtaining the app files
+=======================
+
+Download the main repo and submodules using:
+
+    $ git clone --recurse git@github.com:xmos/sln_voice.git
+    $ cd sln_voice/
+
+
+Building the app
+================
+
+First install and source the XTC version: 15.2.1 tools. You should be able to see something like this:
+
+    $ xcc --version
+    xcc: Build 19-198606c, Oct-25-2022
+    XTC version: 15.2.1
+    Copyright (C) XMOS Limited 2008-2021. All Rights Reserved.
+
+Linux or Mac
+------------
+
+To build for the first time you will need to run cmake to create the make files:
+
+    $ mkdir build
+    $ cd build
+    $ cmake --toolchain ../xmos_cmake_toolchain/xs3a.cmake  ..
+    $ make example_mic_aggregator_tdm -j
+    $ make example_mic_aggregator_usb -j
+
+Following initial cmake build, as long as you don't add new source files, you may just type:
+
+    $ make example_mic_aggregator_tdm -j
+    $ make example_mic_aggregator_usb -j
+
+If you add new source files you will need to run the `cmake` step again.
+
+Windows
+-------
+
+It is highly recommended to use `Ninja` as the make system under cmake. Not only is it a lot faster
+than MSVC `nmake`, it also works around an issue where certain path names may cause an issue with the XMOS compiler under windows.
+
+To install Ninja, follow these steps:
+
+- Download `ninja.exe` from https://github.com/ninja-build/ninja/releases. This firmware has been tested with Ninja version v1.11.1
+- Ensure Ninja is on the command line path. You can add to the path permenantly by following these steps https://www.computerhope.com/issues/ch000549.htm. Alternatively you may set the path in the current command line session using something like `set PATH=%PATH%;C:\Users\xmos\utils\ninja`
```

Comment:
permenantly -> permanently (I noticed this typo only because I re-used this line)

Reply:
Nice catch. I found another issue whilst spell checking too!